### ISSUE TITLE
Music Override Causing Some Scenes To Crash

### ISSUE THREAD

**Issue by Nato1ess** [bunfan/beat-banger-public#203]
### Discord Tag

@Natoless

### Issue Description

When I try to enter a shop with any of the non-default title music equipped the game crashes. 

I think this is the crash log.

### OS

Windows

### Version

3.34 dev1 (Steam)

### Relevant log output

```shell
Godot Engine v4.2.stable.official.46dc27791 - https://godotengine.org
OpenGL API 3.3.14831 Core Profile Forward-Compatible Context 21.5.2 27.20.20903.8001 - Compatibility - Using Device: ATI Technologies Inc. - AMD Radeon R7 Graphics
 
17:20:06: INFO Loading Save Data...
17:20:06: INFO Validating Save...
17:20:06: INFO Saving Game...
17:20:06: INFO Loading User Settings...
USER ERROR: Unable to convert array index 0 from 'Nil' to 'String'.
   at: assign (core/variant/array.cpp:245)
17:20:06: INFO Validating Keybinds...
17:20:06: INFO Loading Keybinds...
WARNING: AnimationMixer: 'RESET', Value Track: '.:default_color' has different update modes between some animations which may be blended together. Blending prioritizes UpdateMode.UPDATE_CONTINUOUS, so the process treats UpdateMode.UPDATE_DISCRETE as UpdateMode.UPDATE_CONTINUOUS with InterpolationType.INTERPOLATION_NEAREST.
   at: _update_caches (scene/animation/animation_mixer.cpp:889)
17:20:06: INFO OS Name : Windows
17:20:06: INFO Game Version : 3.34 - dev1
17:20:14: INFO Loading Scene: res://src/scenes/main_menu.tscn
17:20:14: INFO Loading Save Data...
17:20:14: INFO Validating Save...
17:20:14: INFO Saving Game...
17:20:16: INFO Loading Scene: res://src/scenes/shop.tscn
17:20:16: INFO Loading Scene: res://src/scenes/main_menu.tscn
17:20:17: INFO Loading Save Data...
17:20:17: INFO Validating Save...
17:20:17: INFO Saving Game...
17:20:18: INFO Loading Scene: res://src/scenes/shop.tscn
17:20:18: INFO Loading Scene: res://src/scenes/main_menu.tscn
17:20:18: INFO Loading Save Data...
17:20:18: INFO Validating Save...
17:20:18: INFO Saving Game...
17:20:19: INFO Loading Scene: res://src/scenes/extras.tscn
17:20:21: INFO Loading Scene: res://src/scenes/sound_test.tscn
17:20:22: Now Playing: Pre Alpha Theme
17:20:23: INFO Loading Scene: res://src/scenes/extras.tscn
17:20:24: INFO Loading Scene: res://src/scenes/main_menu.tscn
17:20:24: INFO Loading Save Data...
17:20:24: INFO Validating Save...
17:20:24: INFO Saving Game...
17:20:25: INFO Loading Scene: res://src/scenes/shop.tscn
USER ERROR: Can't create Tween when not inside scene tree.
   at: create_tween (scene/main/node.cpp:2279)
```

**Comment by Stefan-Mitric** [comment]
Update: It only crashes on the "I Didn't Sleep" Record.

**Comment by komdog** [comment]
Fixed in 3.34
